Common/UnitDefinitions/TemperatureChangeRate.json

```diff
@@ -35,15 +35,89 @@
     {
       "SingularName": "DegreeKelvinPerMinute",
       "PluralName": "DegreesKelvinPerMinute",
-      "FromUnitToBaseFunc": "({x} / 60) - 273.15",
-      "FromBaseToUnitFunc": "({x} + 273.15) * 60",
+      "FromUnitToBaseFunc": "{x} / 60",
+      "FromBaseToUnitFunc": "{x} * 60",
```

Comment:
Good catch, I missed this when I last reviewed this by checking google, which gives the wrong answer:
https://www.google.com/search?q=DegreesKelvinPerMinute+In+1+DegreeCelsiusPerSecond&rlz=1C1ONGR_enNO1008NO1008&oq=DegreesKelvinPerMinute+In+1+DegreeCelsiusPerSecond&gs_lcrp=EgZjaHJvbWUqBggAEEUYOzIGCAAQRRg70gEINTE2NmowajeoAgCwAgA&sourceid=chrome&ie=UTF-8

It totally makes sense for kelvin and celsius rate of change to be equal.

Reply:
@angularsen thanks for merging my PR. Do you know (roughly) when you will release a new Nuget package?

Reply:
Thanks for the reminder, I got busy and forgot it (again). Maybe it is my age 🤔

Reply:
ahhh thank you for catching my mistake, as well as the links explaining the correct calculation!!